src/main/java/com/cburch/logisim/std/io/RgbArrayRowScanningHdlGeneratorFactory.java

```diff
@@ -132,7 +111,7 @@ public LineBuffer getModuleFunctionality(Netlist theNetlist, AttributeSet attrs)
                    :  {{insB}}[{{nrOfColumns}} * s_rowCounterReg + i];
              end
           endgenerate" +
-          """);
+          """).empty();
```

Comment:
Why not adding empty line to the text block directly?

Reply:
This has to do with the ```.split("\n")``` operation, it will "remove" the last empty line, hence it is a "work around".